### DISCUSSION TITLE
No module named fforces

### DISCUSSION THREAD

**Discussion by wenlingren** [SSCHAcode/python-sscha#131] [Q&A] [answered]
When I work with toturial 3 using 'import fforces as ff'. there is a error 'No module named 'fforces''. I think I install sscha and cellconstructor properly. And I can run toturial 2 and 1 correctly. So where can I get the moudle 'fforces' and how can I install it.

**Comment by diegomartinez2** [comment, marked as answer]
Hi, the fforces is a toy model used in the tutorial is downloaded from the SSCHA repository; Basically:
`git clone https://github.com/SSCHAcode/F3ToyModel`

`cd F3ToyModel`

`python setup.py install`

Or copying from the installation part of the tutorial (probably you have an older version that didn't have this):

F3ToyModel installation
-----------------------

F3ToyModel is a force-field that can mimic the physics of ferroelectric transitions in FCC lattices. All preresquisites are met with the SSCHA installation.

The code for this force-field can be downloaded from the SSCHA github `here <https://github.com/SSCHAcode/F3ToyModel>`

Now enter the F3ToyModel directory and install with:

                  python setup.py install
